po/de.po

```diff
@@ -7530,7 +7530,7 @@ msgstr "auto"
 #: ../src/gui/preferences.c:851 ../src/gui/preferences.c:957
 msgctxt "preferences"
 msgid "import..."
-msgstr "importieren"
+msgstr "importieren…"
```

Comment:
The dots do not make any sense. Neither in the English original nor in the German translation. They just claim precious UI space.